File: 2°SEMESTRE/UC8-prog_orient_obj/JAVA/aula_java/src/lista_exercicios_02/ex13_jow.java
//Crie um programa que leia um número inteiro positivo par N e imprima todos os números pares de 0 até N em ordem crescente.
package lista_exercicios_02;

import java.util.ArrayList;
import java.util.List;
import java.util.Scanner;

public class ex13_jow {
           public static void main(String[] args){
        Scanner scan = new Scanner(System.in);
        System.out.println("Digite um numero: ");
        double num = scan.nextDouble();
        double cont=0;
        List<Double> num_pares = new ArrayList<>();
        while(num+1>cont){
            if((cont)%2==0){
                num_pares.add(cont);
            }
            cont++; 
        }
        System.out.println("Os numeros pares de 0 a 100 são:\n"+num_pares);

        scan.close();
    }
}

File: ListaPython-to-22-04-2025/ex13-jow.py
#Verifique se um número é positivo e maior que 100.
num=int(input('Digite um numero: '))
if(num>0 and num>100):
    print('O numero digitado é positivo e maior que 100')
elif(num>0 and num<=100):
    print('O numero digitado é positovo, mas menor que 100')
else:
    print('O numero digitado é neutro ou negativo')
